main.go

```diff
@@ -269,3 +287,28 @@ func validateGitHubResponse(res *github.Response, err error) error {
 	}
 	return nil
 }
+
+func getProjectCardsFromIssue(ctx context.Context, client *github.Client, issue *github.Issue, owner string, repo string) ([]*github.ProjectCard, error) {
+	cards := []*github.ProjectCard{}
+	// TODO: Pagination, but let's assume any project will be in the first set of results.
+	events, resp, err := client.Issues.ListIssueEvents(ctx, owner, repo, *issue.Number, nil)
+	err = validateGitHubResponse(resp, err)
+	if err != nil {
+		return cards, err
+	}
+	for _, event := range events {
+		if event.ProjectCard != nil {
+			cards = append(cards, event.ProjectCard)
+		}
+	}
+	return cards, nil
+}
+
+func moveCardInProject(ctx context.Context, client *github.Client, columnID int64, card *github.ProjectCard) error {
+	resp, err := client.Projects.MoveProjectCard(ctx, *card.ID, &github.ProjectCardMoveOptions{ColumnID: columnID, Position: "bottom"})
```

Comment:
Let's put the new cards at the top of the specified column.

```suggestion
	resp, err := client.Projects.MoveProjectCard(ctx, *card.ID, &github.ProjectCardMoveOptions{ColumnID: columnID, Position: "top"})
```

Reply:
wfm - I'll rebase into the original commit